.github/workflows/xk6.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           cd .github/workflows/xk6-tests
           # Temporary hack to ignore the v0.33.0 tag change
           export GONOSUMDB="go.k6.io/k6"
-          go get go.k6.io/xk6/cmd/xk6@master
+          go install go.k6.io/xk6/cmd/xk6@latest
```

Comment:
Let's split this change into a separate commit, or better yet if you can submit it as a separate PR so that we can merge it quickly and fix it for other PRs.

Reply:
I would prefer to split this into a dedicated PR so we can avoid the dependency on this PR.

Reply:
There is now a separate PR: https://github.com/grafana/k6/pull/2153

Reply:
Removed this from this PR.